ng/src/web/pages/portlists/component.js

```diff
@@ -299,6 +302,7 @@ class PortListComponent extends React.Component {
             {portListDialogVisible &&
               <PortListsDialog
                 comment={comment}
+                id={id}
                 name={name}
                 port_list={port_list}
```

Comment:
I guess port_list isn't used in the PortListDialog anymore and can be safely removed

Reply:
It's still in use to check whether the dialog is_edit, and the newrangeicon (ll. 70) seems to get more data from the port_list than just the id.